### PR TITLE
feat(client): change camper badges layout in profile

### PR DESCRIPTION
**Pull request by Sboonny**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Ref #48234

```console
db.user.findOneAndUpdate({ username: 'certifieduser' }, { $set: { isDonating: true } })
```

### CODE REVIEW

File: client/src/components/profile/components/camper.tsx [hunk 1]
```diff
@@ -100,26 +100,33 @@ function Camper({
...
+          <dd>{t('donate.supporter')}</dd>
+          {isTopContributor && (
+            <>
+              <FontAwesomeIcon icon={faAward} />
+              <dt className='yearsTopContributor'>
+                <Link to={t('links:top-contributors')}>
+                  {t('profile.contributor')}
+                </Link>
+              </dt>
+              <dd>{t('donate.top-contributor')}</dd>
```
Comment (naomi-lgbt) [inline client/src/components/profile/components/camper.tsx:122]: Why is the top contributor logic within the isDonating logic?

Top contributors are based on contributions to GitHub/Forum/News etc - not financial contributions.

Comment (Sboonny) [reply client/src/components/profile/components/camper.tsx:122]: the logic will change when the PR is ready for review, I have added it lazily to focus on adding the CSS that will sort the layout

Comment (naomi-lgbt) [reply client/src/components/profile/components/camper.tsx:122]: Sounds good - thanks for clarifying 💜

**Comment by Sboonny** [comment]
I have discussed the mockup with Ahmad, and a new one will be made.

I am opening this for review, because refactoring the camper element into components and for each prop will make changing them in the future simpler.

**Pull request closed without merging**